csss-site/src/events/templates/events/fall_hacks2020.html

```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>

```

Comment:
any particular reason this isn't following the template design of our website?

Reply:
No.

Reply:
then please follow the template design of the website